.github/dependabot.yml

```diff
@@ -0,0 +1,36 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
```

Comment:
This is my ignorance showing,  us what uses pip?

Reply:
mkdocs